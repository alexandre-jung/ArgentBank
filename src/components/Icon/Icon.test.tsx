import { render } from '@testing-library/react';

import { Icon } from '.';

describe('test', () => {
  it('should render', () => {
    const { container } = render(
      <Icon name="user" className="test-class" data-testid="bonjour" />
    );
    expect(container).toMatchSnapshot();
  });
});
