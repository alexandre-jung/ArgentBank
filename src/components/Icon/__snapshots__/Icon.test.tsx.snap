// Vitest Snapshot v1

exports[`test > should render 1`] = `
<div>
  <svg
    class="test-class"
    data-testid="bonjour"
    fill="currentColor"
    height="24"
    viewBox="0 0 24 24"
    width="24"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M20.397 17.933c-.196-1.384-.587-2.533-1.172-3.449-.584-.915-1.408-1.444-2.47-1.587a6.36 6.36 0 0 1-2.137 1.547A6.325 6.325 0 0 1 12 15c-.92 0-1.792-.185-2.618-.556a6.36 6.36 0 0 1-2.136-1.547c-1.063.143-1.887.672-2.471 1.587-.585.916-.976 2.065-1.172 3.449.946 1.34 2.156 2.4 3.63 3.18A10.022 10.022 0 0 0 12 22.287c1.705 0 3.295-.39 4.768-1.172a10.266 10.266 0 0 0 3.63-3.181Zm-3.254-9.362c0-1.42-.502-2.631-1.507-3.636C14.632 3.931 13.42 3.43 12 3.43s-2.632.502-3.636 1.506C7.359 5.94 6.857 7.152 6.857 8.571c0 1.42.502 2.632 1.507 3.637 1.004 1.004 2.216 1.506 3.636 1.506s2.632-.502 3.636-1.506c1.005-1.005 1.507-2.217 1.507-3.637ZM24 12c0 1.625-.317 3.176-.95 4.654-.635 1.478-1.485 2.755-2.552 3.83-1.067 1.076-2.342 1.931-3.824 2.565-1.482.634-3.04.951-4.674.951-1.625 0-3.179-.317-4.66-.95-1.483-.635-2.76-1.488-3.831-2.559-1.071-1.071-1.924-2.348-2.558-3.83A11.73 11.73 0 0 1 0 12c0-1.625.317-3.179.95-4.66.635-1.483 1.488-2.76 2.559-3.831C4.58 2.438 5.857 1.585 7.339.95A11.73 11.73 0 0 1 12 0c1.625 0 3.179.317 4.66.95 1.483.635 2.76 1.488 3.831 2.559 1.072 1.071 1.924 2.348 2.558 3.83C23.683 8.821 24 10.375 24 12Z"
    />
  </svg>
</div>
`;
